tests/unit/data/package_requests/confluent_kafka_direct.json

```diff
@@ -0,0 +1,114 @@
+        "package.argoproj.io/installer": "argopm",
+        "package.argoproj.io/name": "a-t-ratlans-l-a-s-hkafka-confluent-cloud",
+        "package.argoproj.io/registry": "httpsc-o-l-o-ns-l-a-s-hs-l-a-s-hpackages.atlan.com",
+        "orchestration.atlan.com/default-confluent-kafka-123456": "true",
+        "orchestration.atlan.com/atlan-ui": "true"
+      },
+      "name": "atlan-kafka-confluent-cloud-123456",
+      "namespace": "default"
+    },
+    "spec": {
+      "entrypoint": "main",
+      "templates": [
+        {
+          "name": "main",
+          "dag": {
+            "tasks": [
+              {
+                "name": "run",
+                "arguments": {
+                  "parameters": [
+                    {
+                      "name": "extraction-method",
+                      "value": "direct"
+                    },
+                    {
+                      "name": "skip-internal-topics",
+                      "value": "false"
+                    },
+                    {
+                      "name": "include-filter",
+                      "value": ".*_TEST"
+                    },
+                    {
+                      "name": "credential-guid",
+                      "value": "{{credentialGuid}}"
+                    },
+                    {
+                      "name": "connection",
+                      "value": "{\"typeName\": \"Connection\", \"attributes\": {\"qualifiedName\": \"default/confluent-kafka/123456\", \"name\": \"test-conf-kafka-direct-conn\", \"adminUsers\": [], \"adminGroups\": [], \"connectorName\": \"confluent-kafka\", \"isDiscoverable\": true, \"isEditable\": false, \"adminRoles\": [\"admin-guid-1234\"], \"category\": \"eventbus\", \"allowQuery\": false, \"allowQueryPreview\": false, \"rowLimit\": 0, \"defaultCredentialGuid\": \"{{credentialGuid}}\", \"sourceLogo\": \"https://cdn.confluent.io/wp-content/uploads/apache-kafka-icon-2021-e1638496305992.jpg\"}, \"guid\": \"-123456123456\"}"
+                    },
+                    {
+                      "name": "publish-mode",
+                      "value": "production"
+                    },
+                    {
+                      "name": "atlas-auth-type",
+                      "value": "internal"
+                    }
+                  ]
+                },
+                "templateRef": {
+                  "name": "atlan-kafka-confluent-cloud",
+                  "template": "main",
+                  "clusterScope": true
+                }
+              }
+            ]
+          }
+        }
+      ],
+      "workflowMetadata": {
+        "annotations": {
+          "package.argoproj.io/name": "@atlan/kafka-confluent-cloud"
+        }
+      }
+    },
+    "payload": [
+      {
+        "parameter": "credentialGuid",
+        "type": "credential",
+        "body": {
+          "name": "default-confluent-kafka-123456-0",
+          "host": "test-bootstrap-server",
```

Comment:
Are you sure the host doesn't include the `:9092` at the end? Thought I had seen that this is in both the end of the host and in the separate port field...

Reply:
Thanks for catching that! I missed it in the tests 👍